tests/test_build.py

```diff
@@ -281,6 +281,28 @@ def test_sidebars_level2(sphinx_build_factory, file_regression):
     file_regression.check(sidebar.prettify(), extension=".html")
 
 
+def test_sidebars_show_nav_level0_index(sphinx_build_factory, file_regression):
+    """"""
```

Comment:
These are both empty docstrings, can you add a one-sentence explanation of each test?